KS.Fiks.IO.Arkiv.Client/Schema/sok.xsd

```diff
@@ -204,12 +205,45 @@
         </xs:restriction>
     </xs:simpleType>
 
-    <xs:simpleType name="respons_filter">
+    <xs:simpleType name="respons_type">
         <xs:restriction base="xs:string">
             <xs:enumeration value="noekler"/>
             <xs:enumeration value="minimum"/>
             <xs:enumeration value="utvidet"/>
         </xs:restriction>
     </xs:simpleType>
+
+    <xs:simpleType name="inkluder">
+        <xs:restriction base="xs:string">
+            <xs:enumeration value="oppbevaringssted"/>
+            <xs:enumeration value="arkivskaper"/>
+            <xs:enumeration value="klassifikasjonssystem"/>
+            <xs:enumeration value="arkiv"/>
+            <xs:enumeration value="arkivdel"/>
+            <xs:enumeration value="mappe"/>
+            <xs:enumeration value="registrering"/>
+            <xs:enumeration value="klasse"/>
+            <xs:enumeration value="noekkelord"/>
+            <xs:enumeration value="kryssreferanse"/>
+            <xs:enumeration value="referanseArkivdel"/>
+            <xs:enumeration value="part"/>
+            <xs:enumeration value="merknad"/>
+            <xs:enumeration value="referanseArkivdel"/>
+            <xs:enumeration value="presedens"/>
+            <xs:enumeration value="postadresse"/>
```

Comment:
Postadresse og telefonnummer er attributter på part og korrespondansepart. Det er sjelden mer enn en av hver og dette er dessuten på et undernivå. Tar man med part, tas all adresseinfo med for disse. Fjern disse to som inkluder.

Reply:
Tatt bort